### PR TITLE
need help

### PR DESCRIPTION
**Pull request by vadim9999**

### CODE REVIEW

File: src/Layout3View.js [hunk 1]
```diff
@@ -63,7 +65,7 @@ const styles = StyleSheet.create({
   </View>
 </View> */}
 
-const MyDocument = () => (
+const Layout3View = () => (
```
Comment (atherdon) [inline src/Layout3View.js]: i think i got it. see at function arguments? we did't pass `data` as props.

i assume if we'll change it to `Layout3View = ({ data }) => (` it maybe will work

File: src/RecipeLayoutView.js [hunk 2]
```diff
@@ -63,7 +63,7 @@ const styles = StyleSheet.create({
   </View>
 </View> */}
 
-const MyDocument = () => (
+const RecipeLayoutView = () => (
```
Comment (atherdon) [inline src/RecipeLayoutView.js]: same as previous comment

File: src/Layout3View.js (hunk 1 above)
Comment (vadim9999) [reply src/Layout3View.js]: const Layout3View = ({ data }) => ( 
doesn't work

Comment (atherdon) [reply src/Layout3View.js]: ok, commit your update and i'll take a look

Comment (atherdon) [reply src/Layout3View.js]: don't worry i'm fetching your forked repository

**Comment by atherdon** [comment]
in short - we just have an issue in import :)
but it was not easy to catch.
merging

**Pull request merged**